src/Octoshift/Services/ReclaimService.cs

```diff
@@ -217,12 +234,31 @@ private bool HandleInvitationResult(string mannequinUser, string targetUser, Man
         return true;
     }
 
+    /// <summary>
+    /// Processes the reclatimationr results returned by the api, handles any returned errors,
+    /// logs missing data, and logs reclaimation result.
+    /// </summary>
+    /// <param name="mannequinUser"></param>
+    /// <param name="targetUser"></param>
+    /// <param name="mannequin"></param>
+    /// <param name="targetUserId"></param>
+    /// <param name="result"></param>
+    /// <returns>returns true when CSV should continue onto the next user.
+    /// Returns false when iteration through the CSV  should stop and fail fast.</returns>
     private bool HandleReclaimationResult(string mannequinUser, string targetUser, Mannequin mannequin, string targetUserId, ReattributeMannequinToUserResult result)
     {
         if (result.Errors != null)
         {
-            _log.LogError($"Failed to reclaim {mannequinUser} ({mannequin.Id}) to {targetUser} ({targetUserId}): {result.Errors[0].Message}");
-            return false;
+            // Writing as switch statement in anticipation of other errors that will need specific logic
+            switch (result.Errors[0].Message)
+            {
+                case string a when a.Contains("is not an Enterprise Managed Users (EMU) organization"):
```

Comment:
@brianaj This is where I handled the "org is not emu" scenario. I set up a switch statement so we can catch other specific error messages and handle as needed (i.e. provide specific log error as well as whether or not to fail fast (return false) or continue to the next mannequin (return true).